spec/support/date_format_shared_examples.rb

```diff
@@ -0,0 +1,25 @@
+shared_examples :date_format do
```

Comment:
Just want to give you heads up. AirAsia updated their mobile app which now requires BIG ID credential before logging in. I'm also getting HTTPForbidden error using the master branch. I'm still setting up my python `mitmproxy` but those two incidents are highly possibly related.